### PR TITLE
Update the readme about submodules

### PR DESCRIPTION
**Pull request by troydai**
For a clean cloned repository the build will fail if the submodules are not updated. The read me document needs to be updated to inform user to prepare the repo.

@Eilon @davidfowl @muratg Please help review.

### CODE REVIEW

**Comment by davidfowl** [comment]
You can also mention that you can `git clone --recursive`

**Comment by troydai** [comment]
:ok_hand: will add.

**Comment by muratg** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -34,5 +34,21 @@ Active Version           Runtime Architecture Location
...
+
+This repository includes a few submodules. They need to be initialized before build.
+
+To clone a repository and initialize all the submodules you can run:
+
+```
+git clone --recursive
+```
 
+If you have already cloned the repoistory without `--recursive` option, you can run following commands to initialize the submodules:
```
Comment (Eilon) [inline README.md]: Change `you can run following` --> `run the following` (remove `you can` and add `the`). (Don't need a PR for it, just check it in)

**Pull request merged**